src/lua/log.lua

```diff
@@ -453,6 +453,9 @@ local compat_v16 = {
     end;
 }
 
+-- Log registry. It stores loggers, created by log_new, each of them having a
+-- custom name.
+local log_registry = {}
```

Comment:
Let's add a comment why do we need registry. AFAIK it helps us to return same object on `require('log')`.